concourse/scripts/bosh_login.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
```

Comment:
Should we set `-u` as well to prevent this from being run without an `bosh_secrets_file` argument?